pkg/ephemeral/io/feeder.go

```diff
@@ -50,8 +51,63 @@ type AmphoraFeeder struct {
 	carrier AbstractCarrier
 }
 
-// LoadFromSecretStoreAndFeed loads input parameters from Amphora.
+// LoadByTagsAndSecretStoreAndFeed loads input parameters from Amphora based on tag filter.
+func (f *AmphoraFeeder) LoadByTagsAndSecretStoreAndFeed(act *Activation, feedPort string, ctx *CtxConfig) ([]byte, error) {
+
+	f.logger.Debug("Called LoadByTagsAndSecretStoreAndFeed()")
+
+	var secretShareIds []string
+	client := f.conf.AmphoraClient
+
+	// TODO implement pagination
+	params := &amphora.ObjectListRequestParams{
+		Filter:        strings.Join(act.TagFilterParams, ","),
+		PageNumber:    0,
+		PageSize:      0,
+		SortProperty:  "",
```

Comment:
@sbckr is it fine to sort by creation-date here as first step to ensure the order of secrets is the same on all clusters?

Or should we move to having this provided by the user?
-> And what would be the fallback values if the user would not provide values?
-> I'd move this to a separate feature, if it is required, what do you think?

Reply:
@kindlich 
While `Amphora` allows to define multiple tags for filtering, only one can be used for sorting. And as I think it is essential to sort by date to guarantee the same order of the shares across the VPCs, I therefore think it is crucial to build this into the logic.

Reply:
Hmm, however it could also make sense to have sorting customizable to sort e.g. by sensor ID. 
So how about making `creation-date` default and allow the user to override?

Reply:
@sbckr 
I agree that at some point the user should be able to set the sort property (e.g. when uploading a large dataset row-wise in parallel, then a custom tag for sorting may be useful as the timestamps may not give enough information).

My question was more about whether we are fine with hard-coding to `creation-date` for now and have that an enhancement later, or whether we want user-provided sorting part of this PR?

Reply:
I'd personally go for a user provided sort parameter right from the start but am fine with both options actually . `creation-date` should tackle most of the cases anyway, I'd therefore leave it up to you whether you'd like to implement it or someone else should do it ;-)